src/fn/fn-message-box.scss

```diff
@@ -0,0 +1,92 @@
+@import "./fn-settings";
+@import "./fn-mixins";
+@import "./dialog-placeholders";
+
+/*!
+  .fn-message-box+(--active)
+    .fn-message-box__content
+      .fn-message-box__header
+      .fn-message-box__body+(--no-vertical-padding)
+      .fn-message-box__footer
+*/
+
+$block: #{$fn-namespace}-message-box;
+
+$fn-message-box-icon-text-spacing: 0.5rem;
+$fn-message-box-title-icon-text-size: 1rem;
+
+$message-box-states: (
+  "confirmation": ("iconColor": $fn-color-grey-500),
+  "error": ("iconColor": $fn-color-red-500),
+  "success": ("iconColor": $fn-color-green-500),
+  "warning": ("iconColor": $fn-color-orange-400),
+  "information": ("iconColor": $fn-color-blue-500)
```

Comment:
we should stick with the naming used in the design: positive, critical, negative...

Reply:
updated